bluetooth_mesh/test/test_application.py

```diff
@@ -114,6 +114,9 @@ async def introspect(self, service_name, object_path):
         def export(self, object_path, interface):
             pass
 
+        def unexport(self, object_path):
```

Comment:
?

Reply:
Application calls it in `dbus_disconnected` callback `application.py:518`

Reply:
Because `dbus_disconnected` callback hasn't been called, interpreter didn't raised an exception.